practice/src/css/sections/best-place.css

```diff
@@ -1 +1,271 @@
+.wrapper-place {
+  background: url(../../assets/images/banner.webp) no-repeat center;
+  background-size: cover;
+  border-radius: 24px;
+  height: 380px;
+  display: flex;
+}
 
+.place-introduction {
+  display: none;
+}
+
+.tab-nav {
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+}
+
+.tab-list {
+  display: flex;
+  margin: 0;
+  padding-left: 0px;
+  width: fit-content;
+}
+
+.tab-pane {
+  background-color: var(--primary-background-color);
+  border-radius: 0px 20px 20px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 10px;
+  padding: 15px 10px 10px 10px;
+}
+
+.tab-item {
+  padding: 8px 20px;
+  border-radius: 0px 12px 0px 0px;
+  background-color: var(--primary-background-color);
+}
+
+.native {

```

Comment:
Naming convention. You can use tab-item:first-child and tab-item:last-child to style border. Don't style border-radius here